.github/workflows/pdp_cicd.yml

```diff
@@ -0,0 +1,102 @@
+name: Build and Push PDP Docker Image
+on:
+  release:
+    types: [published]
```

Comment:
Maybe we want to execute this also on pushes to main branch, and release docker image with tag=commit hash?

Reply:
Sorry missed that, ok sounds good